messaging-activemq/subsystem/src/main/resources/org/wildfly/extension/messaging/activemq/LocalDescriptions.properties

```diff
@@ -167,8 +167,10 @@ connection-factory.connection-ttl=The connection ttl.
 connection-factory.connectors=Defines the connectors. These are stored in a map by connector name (with an undefined value). It is possible to pass a list of connector names when writing this attribute.
 connection-factory.consumer-max-rate=The consumer max rate.
 connection-factory.consumer-window-size=The consumer window size.
+connection-factory.deserialization-allow-list=A list of class names (separated by whitespaces) that are allowed to be used in serialization of Jakarta Messaging ObjectMessage.
 connection-factory.deserialization-black-list=A list of class names (separated by whitespaces) that are not allowed to be used in serialization of Jakarta Messaging ObjectMessage.
 connection-factory.deserialization-black-list.deprecated=Will be replaced by a differently-named attribute in a later release.
```

Comment:
These .deprecated messages should be updated to reference the replacement attributes.